pkg/compliance/checks/env/env.go

```diff
@@ -29,5 +29,6 @@ type Configuration interface {
 	Hostname() string
 	EtcGroupPath() string
 	NormalizePath(path string) string
+	RelativePath(path string) string
```

Comment:
We might want to add comments for all these here 😄 it's not evident what relative path means, normalize might be hard to understand as well. Maybe we should rename these to `NormalizeToHostRoot` and `RelativeToHostRoot`?

Reply:
I wanted to clarify, but I was thinking these functions may also resolve variables when/if we introduce them.

Reply:
As in expanding env vars? Yeah so then we'll probably have a set of options for that as an argument? I am just struggling a bit with `RelativePath` but `NormalizePath` is also somewhat confusing.